File: 2016-maximum-difference-between-increasing-elements/2016-maximum-difference-between-increasing-elements.java
class Solution {
    public int maximumDifference(int[] nums) {
        int n = nums.length;
        int ans = -1;
        int max = -1;
        for(int i=0;i<n-1;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i]<nums[j]){
                    int temp = nums[j]-nums[i];
                    max = Math.max(max,temp);
                }
            }
            
        }
        return max;
    }
}

File: 2016-maximum-difference-between-increasing-elements/2016-maximum-difference-between-increasing-elements.py
class Solution:
    def maximumDifference(self, nums: List[int]) -> int:
        n = len(nums)
        ans = -1
        for i in range(n):
            for j in range(i+1,n):
                if nums[i] < nums[j]:
                    temp = nums[j] - nums[i]
                    ans = max(temp,ans)
        return ans      